### PR TITLE
Install archive

### PR DESCRIPTION
**Pull request by renukamanavalan**
1. Creates a self extractive archive that installs 
2. Tested install/rollback/cleanup operations

Ready for review

### CODE REVIEW

File: install/sonic/LoM-install.sh [hunk 1]
```diff
@@ -0,0 +1,429 @@
+#! /bin/bash
```
Comment (GouthamBanala) [inline install/sonic/LoM-install.sh:1]: sudo check ? can this be run with sudo ?

File: install/decompress [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#! /bin/bash
```
Comment (GouthamBanala) [inline install/decompress:1]: sudo check ? can this be run without sudo ?

File: install/decompress [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+echo ""
+echo "==========================="
+echo "Self-extracting LoM archive"
+echo "==========================="
+
+export TMPDIR=`mktemp -d /tmp/selfextract.XXXXXX`
+
+ARCHIVE=`awk '/^__ARCHIVE_BELOW__/ {print NR + 1; exit 0; }' $0`
+
+tail -n+$ARCHIVE $0 | tar xzv -C $TMPDIR
```
Comment (GouthamBanala) [inline install/decompress:12]: everytime it overwrites selfextract.XXXXXX. should we delete it before untar?

File: install/decompress (hunk 2 above)
Comment (renukamanavalan) [reply install/decompress:1]: Yes.
Hence added sudo where needed.
Just trying to minimize the ask to FUSE or anyone.

Said that, I need to check if the caller has sudo privileges at the start. I will do.

File: install/decompress (hunk 3 above)
Comment (renukamanavalan) [reply install/decompress:12]: XXXX is replaced by digits that makes it uniique.
I intentionally not remove the dir as that could be used for debugging later 
Being in /tmp folder, it is OK *for now*
Each call creates a new unique dir

File: install/decompress (hunk 2 above)
Comment (renukamanavalan) [reply install/decompress:1]: Done.

File: install/sonic/LoM-install.sh [hunk 4]
```diff
@@ -0,0 +1,443 @@
...
+
+    # Restart installed/upgraded/rolledback service instance.
+    #
+    sudo systemctl restart device-health.service 
+    [[ $? != 0 ]] && { fail "Failed to restart device-health service" ${ERR_RESTART}; }
+
+    # Pause for a minute
+    #
+    echo "Take a 1 min pause to settle down"
+    sleep 1m
```
Comment (GouthamBanala) [inline install/sonic/LoM-install.sh:357]: Isn't 1min large? There is a check for processes start next , can't we have timer based check like at 
https://github.com/GouthamBanala/sonic-device-health/blob/gou_working/lom/integration_test/src/api.py API :: wait_for_lom_services_to_start

Comment (GouthamBanala) [reply install/sonic/LoM-install.sh:357]: From my experience from Integration tests, time is less than min

Comment (renukamanavalan) [reply install/sonic/LoM-install.sh:357]: Better be safe. 
We can't afford to have any false positive.
Think of FUSE pipeline failure.

**Review by GouthamBanala — APPROVED** [review]

**Pull request merged**